packages/devtools_app/lib/main.dart

```diff
@@ -2,103 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:async';
-import 'package:html_shim/html.dart';
-
-import 'package:devtools_app/src/framework/framework_core.dart';
-import 'package:devtools_app/src/main.dart';
-import 'package:devtools_app/src/ui/analytics.dart' as ga;
-import 'package:devtools_app/src/ui/analytics_platform.dart' as ga_platform;
-import 'package:platform_detect/platform_detect.dart';
+import 'src/_app_html.dart' if (dart.library.io) 'src/_app_flutter.dart'
```

Comment:
I'm not confident that we'll be able to make this work on bazel.

My suggestion is to create a second entrypoint, and use that in a flutter_index.html file, leaving the existing entrypoints the same.

Reply:
Creating a separate entry point is fine. In the external workflow we are just missing the hooks to have multiple entrypoints.
_app_html.dart is already the entrypoint you are looking for.

Reply:
We discussed over chat, but sharing here:

`flutter run -t lib/alternate_entrypoint.dart` is how to use a separate entrypoint.

However, all entrypoints will load index.html, which leads to the devtools web app being drawn underneath the devtools flutter web app with the manual entrypoint.